tests/openthread/Makefile

```diff
@@ -12,11 +13,25 @@ RIOTBASE ?= $(CURDIR)/../..
 QUIET ?= 1
 
 USEPKG += openthread
-USEMODULE += openthread_contrib
-USEMODULE += libmbedcrypto
-USEMODULE += libopenthread
-USEMODULE += libopenthread-cli
-USEMODULE += xtimer
+OPENTHREAD_TYPE ?= ftd
+ifeq ($(OPENTHREAD_TYPE),mtd)
+  # MTD: A Minimal Thread Device does not have router functionality
+  # compiled in. As a result, it is not necessary to configure the
+  # routerrole on an MTD. At the same time, an MTD may or may not be sleepy.
+  USEMODULE += openthread-mtd
+  USEMODULE += openthread-cli-mtd
+else
+  # ftd: A Full Thread Device has router functionality compiled in
+  USEMODULE += openthread-ftd
+  USEMODULE += openthread-cli-ftd
+endif
+
+#Define PANID, CHANNEL and UART_BAUDRATE used by default
+OPENTHREAD_PANID ?= 0xbeef
+OPENTHREAD_CHANNEL ?= 26
+
+CFLAGS += -DOPENTHREAD_PANID=${OPENTHREAD_PANID}
```

Comment:
minor: use parenthesis instead of curly braces for accessing variables in Makefiles